docs/serving/samples/cloudevents/cloudevents-dotnet/README.md

```diff
@@ -1,10 +1,3 @@
----
-title: "Cloud Events - C#"
-linkTitle: "C#"
-weight: 1
-type: "docs"
----
-
```

Comment:
Did you mean to drop this?

Reply:
Yes, it looks like other samples, including the go one now has this information in _index.md instead. So I moved it there.